### DISCUSSION TITLE
Ability to add custom route to the notification object

### DISCUSSION THREAD

**Discussion by drennvinn** [directus/directus#19616] [Feature Requests]
### Summary

In order to be able to redirect to a location other than /admin/content, for instance, if we have custom modules or other requirements, it would be great to have the ability to choose a custom redirection if needed.

### Motivation

Enabling custom redirection will empower users to tailor the platform to their specific processes, improving overall efficiency and user experience

### Detailed Design

To support custom routes in Directus notifications, we could modify the payload structure. Specifically, we will replace the "item" and "collection" fields with a new "route" object I think.

### Requirements List

...

### Drawbacks

...

### Alternatives

...

### Adoption Strategy

...

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
